classes/fields/taxonomy.php

```diff
@@ -12,6 +12,7 @@ class PodsField_Taxonomy extends PodsField_Pick {
      * @since 2.0
      */
     public function __construct () {
+	    parent::__construct();
```

Comment:
> this field type just maps to the relationship field

No real use I guess since everywhere in Pods this field is called a relationship field. 
From the looks of it this file is there for backwards compatibility?